docker/py36.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6.10-alpine3.11
+
+WORKDIR /home
+
+COPY requirements_dev.txt .
+
+RUN apk add gcc py-pip python-dev libffi-dev openssl-dev gcc libc-dev make
+
+RUN python -m pip install psutil subprocess32
```

Comment:
It's been a while, but I was thinking we don't need to install `subprocess32` outside of Python 2.7 as it's a backport of the Python 3.2+ version of subprocess.

Reply:
Removed. I'' add a commit with bash installed too